### DISCUSSION TITLE
The map should always be shown north up, until route has been started

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#18089] [Ideas]
Hello!

When you plan a trip, after entering the destination, the app calculates the route and displays the hole route. This should always be done with north pointing upwards, for easing the orientation. The map should not be rotated until the user hits "Start".

I find it extremely inconvenient, when I look at a map that shows my route from east to west, and that route goes from top to bottom of the screen.

**Comment by schubi74** [comment]
I totally agree with you! I am switching profiles to change the routing preferences (online, offline etc.), but NOT to change the orientation of the map. But as the orientation setting for most routing profiles is "by movement", the map is suddenly rotated to some random orientation which only made sense when I finished the last navigation (maybe days ago). That does not make any sense! And it keeps turning around when jitter happens on the GPS signal.

To stay oriented when planning a route I have to apply "North is up" during planning and then choose "by movement" once I started routing. Every commercial navigation system (e.g. Tomtom) does these switchovers automatically. Why can't Osmand handle it the same way, as it already proved to be the most intuitive?
